.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Lint Check
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    name: Run ESLint and Prettier
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18 # Update to Node.js 18 or higher
+      
+
+      - name: Install dependencies
+        run: npm install
```

Comment:
maybe use npm ci instead of npm install 

https://docs.npmjs.com/cli/v11/commands/npm-ci